internal/pkg/aws/acm/acm.go

```diff
@@ -92,5 +107,23 @@ func (a *ACM) findValidAliasesAgainstCert(ctx context.Context, aliases []string,
 			validAliases = append(validAliases, alias)
 		}
 	}
-	return validAliases, nil
+	return validAliases
+}
+
+type errInValidAliasAgainstCert struct {
+	certs         []string
+	alias         string
+	domainsOfCert map[string][]string
+}
+
+func (e *errInValidAliasAgainstCert) Error() string {
+	return fmt.Sprintf("%s is not a valid domain against %s", e.alias, strings.Join(e.certs, ","))
+}
+
+func (e *errInValidAliasAgainstCert) RecommendActions() string {
+	var logMsg string
+	for cert, sans := range e.domainsOfCert {
+		logMsg += fmt.Sprintf("Your imported certificate '%s' protects the following domains:\n%s\n", cert, strings.Join(sans, "\n"))
+	}
```

Comment:
nit: When there are multiple certs, this reads:
```
Your imported cert 'arn:aws:acm:us-west-2:111111111111:certificate/5033ba2b-09b2-4bc0-8dbb-51adf3e12345' protects the following domains:
a.com, b.com
Your imported cert 'arn:aws:acm:us-west-2:111111111111:certificate/5033ba2b-09b2-4bc0-8dbb-51adf3e54321' protects the following domains:
c.com, d.com
```
The repetition *might* sound a little robotic?
What do you think of something like:
```
Please use aliases that are protected by the certificates your imported:
'5033ba2b-09b2-4bc0-8dbb-51adf3e12345': a.com, b.com
'5033ba2b-09b2-4bc0-8dbb-51adf3e54321': c.com, d.com
```

We can take advantage of [`english.Plural`](https://pkg.go.dev/glitch.sh/go/humanize/english#Plural) to use "certificate" (singular) or "certificates" (plural) depending on the number of certs imported.
https://pkg.go.dev/glitch.sh/go/humanize/english#WordSeries